swagger_parser/README.md

```diff
@@ -15,33 +15,62 @@
 - Support for multiple languages (Dart, Kotlin)
 - Web interface at https://carapacik.github.io/swagger_parser
 
-### Run command
+## Installation
 
-To generate boilerplate code, run the `generate` program inside directory where your `pubspec.yaml` file is located:
-```shell
-dart run swagger_parser:generate
-```
+### Add dependency
 
-Add your OpenApi json file configuration to your `pubspec.yaml` or create a new config file called `swagger_parser.yaml`.
-An example of YAML is shown below
+In your pubspec.yaml, add the following dependencies:
 
-### Configure package
 ```yaml
+dependencies:
```

Comment:
It's probably better to remove specific versions of packages, except for swagger_parser. It's just enough not to write anything after the colon.

Reply:
Well, following semver it's not good to let unmapped major compatible versions as a major release could crash the generator due breaking changes in the dependency.